dist/index.js

```diff
@@ -379,13 +379,14 @@ class Action {
         core.summary.addHeading(`Pipeline result: ${report.passed ? "passed" : "failed"}`);
         let tasksPassed = 0;
         let tasksFailed = 0;
+        let tasksSkipped = 0;
         let testsTable = "<table><thead><tr><td colspan=5>Tests</td></tr>"
             + "<tr><td>Action</td><td>Passed 🟢</td><td>Skipped ⚪</td><td>Failed 🔴</td><td>Result</></tr></thead><tbody>";
         let vulnerabilitiesTable = "<table><thead><tr><td colspan=8>Vulnerabilities</td></tr>"
             + "<tr><td>Action</td><td>Minimal</td><td>Low</td><td>Medium</td><td>High</td><td>Criticalℹ️</td><td>Unknown</td>"
             + "<td>Result</td></tr></thead><tbody>";
         for (const task of report.actions) {
-            task.passed ? tasksPassed++ : tasksFailed++;
+            task.passed ? tasksPassed++ : !task.passed ? tasksFailed++ : tasksSkipped++;
```

Comment:
Multiple ternary operators make the code complex and difficult to read.
My suggestion is if you are incrementing `tasksSkipped` instead of getting that value from `executionGraph, I would go back to the if-block.